Assets/Scripts/Battlefield/GridSystem/Tile.cs

```diff
@@ -7,9 +7,16 @@ public class Tile : MonoBehaviour
     
     public GameObject unitOnTile;
     public bool walkable = true;
+    [HideInInspector]
+    public Vector2 coordinates;
 
     public Vector3 GetPos()
     {
         return transform.position;
     }
+
+    public Vector2 GetCoordinates()
```

Comment:
I know this is picky but please don't use Coordinates but something more explicit like GridCoordinates() or the like because GetPos() and GetCoordinates() sound like the same thing in the unity context.